code/modules/mob/mob_defines.dm

```diff
@@ -213,3 +213,8 @@
 
 	/// A mock client, provided by tests and friends
 	var/datum/client_interface/mock_client
+
+	/// Used for tracking last uses of emotes for cooldown purposes
+	var/list/emotes_used
+	var/static/emote_antispam = 0.1 SECONDS
```

Comment:
Doesn't need to be static.

Reply:
Also I'm really not a fan of adding more crap to `/mob`. Do we *really* need per-emote cooldowns?

Reply:
I argued that we don't need per-emote cooldowns, but @PowerfulBacon told me to implement per-emote cooldowns with a varied set of values.